trieste/utils/pareto.py

```diff
@@ -13,12 +13,15 @@
 # limitations under the License.
 """ This module contains functions and classes for Pareto based multi-objective optimization. """
 
+from __future__ import annotations
+
 from typing import Tuple
 
 import tensorflow as tf
 from typing_extensions import Final
 
 from ..type import TensorType
+from .misc import DEFAULTS
 
 
 def non_dominated(observations: TensorType) -> Tuple[TensorType, TensorType]:
```

Comment:
do you mean the outputs of `non_dominated `function?

Reply:
yep

Reply:
The reason is we don't want to make the function slower, and haven't found TF workaround to do it efficiently. so we decide to mention in the docstring that the duplicate points are kept. detail: https://github.com/secondmind-labs/trieste/pull/143#discussion_r575835476 
what do you think?

Reply:
That makes sense. Where is the presence of duplicate points taken into account later on?

Reply:
I am afraid I don't have an answer to that for now. it might be in the test of an acquisition function algorithm when duplicate points arise frequently. but I don't have any concrete example for that for now.